riotfile.py

```diff
@@ -1159,5 +1159,16 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ]
             },
         ),
+        Venv(
+            name="aredis",
+            pys=select_pys("3.6"),
+            command="pytest {cmdargs} tests/contrib/aredis",
+            pkgs={
+                "pytest-asyncio": latest,
+                "aredis": [
+                    latest,
```

Comment:
which versions are we targeting?

we should probably test with at least:

```
"aredis": [
    "~=1.1.0",
    latest,
]
```

"~=1.0.0" seems nice to have, but is quite old so maybe depends on if it "just works"?